game.js

```diff
@@ -12,6 +12,20 @@ function randTime(min, max) {
   return Math.random() * (max - min) + min;
 }
 
+let visibleTime = function(score) {
```

Comment:
There's no docblock for this function. Also seems like almost every other function is a fat arrow, would suggest making them all fat arrow for consistency sake.

We want the end result code to look like it was written by one person.